packages/js/settings-editor/src/components/routes/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { createElement, useEffect } from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
+/* eslint-disable @woocommerce/dependency-group */
+// @ts-ignore No types for this exist yet.
+import { unlock } from '@wordpress/edit-site/build-module/lock-unlock';
+// @ts-ignore No types for this exist yet.
+import { store as editSiteStore } from '@wordpress/edit-site/build-module/store';
+/* eslint-enable @woocommerce/dependency-group */
+
+/**
+ * Internal dependencies
+ */
+import { Sidebar } from '../sidebar';
+import { LegacyContent } from '../../legacy';
+import { getSettingsSectionPath } from './utils';
+
+const settingsData: SettingsData = window.wcSettings?.admin?.settingsData || [];
+
+const routes = Object.values( settingsData ).map( ( settingsPage ) => {
```

Comment:
Do we need to include modern page routes (e.g. register the routes via JS filter) here?

Reply:
Probably yes. Here is where I need help to be aware of the implementation details. Looking 👀

Reply:
I've reorganized the files